sms/src/main/resources/webapp/js/directives.js

```diff
@@ -20,4 +21,18 @@
             }
         };
     });
+
+    // TODO: Can be removed after common file upload directive work again. (MOTECH-2265)
```

Comment:
MOTECH-2265 is about the period modal not working after minification. What does this have to do with a 'common file upload drective'? What is the 'common upload directive' anyway, aren't we adding this in this ticket?